packages/build/src/core/severity.js

```diff
@@ -0,0 +1,19 @@
+// Used for example as exit codes.
+// 1|2|3 indicate whether this was a user|plugin|system error.
+const getSeverity = function(severity = FALLBACK_SEVERITY) {
+  const severityCode = severity in SEVERITY_CODES ? SEVERITY_CODES[severity] : SEVERITY_CODES[FALLBACK_SEVERITY]
```

Comment:
If we're using a default parameter, I don't think we need a ternary here. Something like this would work: 
```
const severityCode = SEVERITY_CODES[severity]
```

Reply:
`severity` could contain a typo, e.g. `warn` instead of `warning`, in which case the default parameter would not apply, but `severityCode` would end up being `undefined`.